### PR TITLE
add-population types

### PR DESCRIPTION
**Pull request by allandegnan**
### What

Describe what you have changed and why.

### How to review

Describe the steps required to test the changes.

### Who can review

Describe who worked on the changes, so that other people can review.

### CODE REVIEW

File: src/js/dp/search.js [hunk 1]
```diff
@@ -240,6 +239,97 @@ if (searchContainer) {
...
+      theChildren.map((item) => {
+        item.addEventListener("change", async (e) => {
+          switchTopicFilterCheckbox([
+            { isChecked: e.target.checked, topics: e.target.value, strParamType: 'population_types'},
+          ]);
+          topicFilter.checked= theChildren.some((x) => x.checked);
+  
+          // Google Tag Manager
+          gtmDataLayerPush({
+            'event': 'SubTopics-Filter',
```
Comment (NoelleDL) [inline src/js/dp/search.js:280]: Line 280 & 266, `event` needs to be updated for population types

File: src/js/dp/search.js [hunk 2]
```diff
@@ -240,6 +239,97 @@ if (searchContainer) {
     })
   });
 
+    // create listeners for topic filter checkboxes
```
Comment (NoelleDL) [inline src/js/dp/search.js:242]: comments should updated for population types/dimensions

File: src/js/dp/search.js (hunk 1 above)
Comment (NoelleDL) [inline src/js/dp/search.js:280]: all the events should be updated as well

File: src/js/dp/search.js [hunk 3]
```diff
@@ -240,6 +239,97 @@ if (searchContainer) {
...
+        gtmDataLayerPush({
+          'event': 'Topic-Filter',
+          'filter-by': e.target.dataset.gtmLabel,
+          'selected': e.target.checked ? 'selected' : 'unselected'
+        });
+      });
+      theChildren.map((item) => {
+        item.addEventListener("change", async (e) => {
+          switchTopicFilterCheckbox([
+            { isChecked: e.target.checked, topics: e.target.value, strParamType: 'topics'},
```
Comment (NoelleDL) [inline src/js/dp/search.js:319]: believe this should be `strParamType: dimensions`

**Comment by allandegnan** [comment]
hi @NoelleDL 

Thanks for the comments, I've pushed another version where I fixed the tag manager, comments, and killed some non-required code.

**Review by NoelleDL — APPROVED** [review]

**Pull request merged**